src/Illuminate/Pipeline/Pipeline.php

```diff
@@ -142,4 +144,22 @@ protected function getInitialSlice(Closure $destination)
 		};
 	}
 
+	/**
+	 * Parse full pipe string to get name and parameters
+	 *
+	 * @param string $pipe
+	 * @return array
+	 */
+	protected  function parsePipeString($pipe)
+	{
+		list($name, $parameters) = array_pad(explode(':', $pipe, 2), 2, []);
+
+		if (is_string($parameters))
+		{
+			$parameters = explode(',', $parameters);
```

Comment:
Interesting idea, although spaces would work with the current implementation (without quotes). The only thing parameters can't contain is a comma.

@taylorotwell What do you think?